### PR TITLE
adding banana, goodbye world

### PR DESCRIPTION
**Pull request by La-di-da**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # hello-world
 This is a read-me
 
-John saw a man on the mountain with a telescope.
-
+John saw a cow on the mountain with a telescope.
```
Comment (La-di-da) [inline README.md]: let's pretend this is python `print("hello world man")`

Comment (jarumihooi) [inline README.md]: Back to "cow"->"man"

File: banana.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+Bananas are stuck in my head.
```
Comment (jarumihooi) [inline banana.txt]: `"Bananas are tasty"`

File: README.md (hunk 1 above)
Comment (La-di-da) [reply README.md]: no. I want this to be cow because cows have more to do with the solar system.

File: banana.txt (hunk 2 above)
Comment (La-di-da) [reply banana.txt]: I think a great idea.

File: README.md (hunk 1 above)
Comment (La-di-da) [reply README.md]: Okay fine I will change it.

File: banana.txt (hunk 2 above)
Comment (jarumihooi) [reply banana.txt]: Actually change my mind, please revert to "stuck in my head"

File: README.md (hunk 1 above)
Comment (jarumihooi) [reply README.md]: Cool.

**Pull request merged**